public/data/hailer24.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "Anmol Bhutani",
+  "type": "personal",
+  "bio": "Just trying to git gud",
+  "avatar": "https://github.com/hailer24.png",
+  "links": [
+    {
+      "name": "Connect with me on GitHub",
+      "url": "https://github.com/hailer24",
+      "icon": "github"
+    },
+    {
+      "name": "Conenct with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/anmol-bhutani-b132931a5/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Take a look at my portfolio",
+      "url": "https://mole24.netlify.app/",
+      "icon": "Link"
+    },
+    {
+      "name": "check my Codeforces account",
```

I think it'd look better if you capitalize `c` here 😅 what do you think? Thanks.
```suggestion
      "name": "Check my Codeforces account",
```